src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -44,6 +45,10 @@
 
 @RobotPreferencesLayout(groupName = "Elevator", row = 1, column = 0, width = 1, height = 2)
 public class Elevator extends SubsystemBase implements ActiveSubsystem, ShuffleboardProducer {
+  private static final double POSITION_TOLERANCE = 0.01;
+  private static final double POSITION_ERROR_MARGIN = 0.1; // meters
```

Comment:
10cm seems too lenient IMHO.
```suggestion
  private static final double POSITION_ERROR_MARGIN = 0.05; // meters
```